package.json

```diff
@@ -64,6 +65,7 @@
     "eslint-plugin-prettier": "^3.0.1",
     "eslint-plugin-react": "^7.12.4",
     "eslint-plugin-react-hooks": "^1.6.0",
+    "faker": "^4.1.0",
```

Comment:
Question: do you need both `faker` and `chance`? Just want to make sure this isn't a leftover.

Reply:
Actually, it might be the other way around: change is the unnecessary dependency here.

Reply:
Yeap, leftover. removing